CONTRIBUTING.rst

```diff
@@ -474,7 +474,7 @@ created the following will occur:
 
     .. code:: bash
 
-       git tag MAJOR.MINOR.0
+       git tag v`python -c "import pyvista as pv; print(pv.__version__)"`
```

Comment:
I think `$()` is usually preferred over backticks, partly because escaping hell doesn't apply with it. Not that this is nested. Just a weak suggestion:
```suggestion
       git tag v$(python -c "import pyvista as pv; print(pv.__version__)")
```